### PR TITLE
feat: adding configuration for details in InfoBox

### PR DESCRIPTION
**Pull request by SophiaSales**
# Summary of Changes
unnamed (1) unnamed (2)

Adding configuration to make InfoBox details optional, as the error message is not clear to the user.

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by bryce-mcmath — DISMISSED** [review]

**Comment by bryce-mcmath** [comment]
@SophiaSales please update this PR with proper DCO sign off, otherwise looks great!

File: packages/legacy/core/App/components/misc/InfoBox.tsx [hunk 1]
```diff
@@ -207,7 +210,7 @@ const InfoBox: React.FC<BifoldErrorProps> = ({
               {showDetails ? message : description}
             </Text>
           )}
-          {message && !showDetails && (
+          {message && !showDetails && showDetailsInfo && (
```
Comment (thiagoromanos) [inline packages/legacy/core/App/components/misc/InfoBox.tsx]: If this configuration is undefined, it should be considered true.

**Review by thiagoromanos — CHANGES_REQUESTED** [review]
Make the config true by default. Check the `InfoBox` file

**Review by thiagoromanos — APPROVED** [review]
Looks good now. :+1:

**Pull request merged**